mod-data-export/src/test/resources/domain/dataexport/features/mapping-profiles.feature

```diff
@@ -83,3 +83,51 @@ Feature: Test mapping profiles
     Then status 403
     And match response contains 'Deletion of default mapping profile is forbidden'
 
+  Scenario: should return UnprocessableEntity response when post mapping profile with invalid transformations - invalid tag
+    Given path 'data-export/mapping-profiles'
+    * print read('classpath:samples/mapping-profile/mp-transformation_invalid_tag.json')
+    And def invalidMappingProfile = read('classpath:samples/mapping-profile/mp-transformation_invalid_tag.json')
+    And request invalidMappingProfile
+    When method POST
+    Then status 422

```

Comment:
Could you add an assertion that 422 was due to `invalid tag` but not due to an another reason? 

If yes, please update all similar tests.

Reply:
There a simple regex that doesn't allow to pass some invalid values and the error message is common for all cases, so I cannot add a check that the invalid tag led to the error.
![error exmple](https://user-images.githubusercontent.com/60663044/118942358-4ecaad00-b95b-11eb-89f4-be41dd1fa665.PNG)
![reason](https://user-images.githubusercontent.com/60663044/118942644-98b39300-b95b-11eb-943a-a9e65a634f17.PNG)

Reply:
Can you verify `value` at least. I assume it has different value depending on the case?